pkg/system/env.go

```diff
@@ -77,6 +77,11 @@ var configOptions = []ConfigOption{
 		DefaultValue: buildah.OCI,
 		OSEnv:        BuildahFormatConfigKey,
 	},
+	{
+		Key:         BuildahLogLevelConfigKey,
+		Description: "the log level to be used in buildah modules",
```

Comment:
fs.StringVar(&globalFlagResults.LogLevel, logLevel, "warn", `The log level to be used. Either "trace", "debug", "info", "warn", "error", "fatal", or "panic".`)

add this description?